src/main/kotlin/io/github/rybalkinsd/kohttp/ext/StringExt.kt

```diff
@@ -61,8 +61,19 @@ fun String.httpGet(client: Call.Factory = defaultHttpClient): Response =
  * @since 0.1.0
  * @author sergey
  */
-fun String.asyncHttpGet(client: Call.Factory = defaultHttpClient): Deferred<Response> =
-    GlobalScope.async(context = Unconfined) {
-        client.suspendCall(Request.Builder().url(this@asyncHttpGet).build())
+fun String.httpGetAsync(client: Call.Factory = defaultHttpClient): Deferred<Response> =
+    GlobalScope.async(context = Dispatchers.Unconfined) {
+        client.call(Request.Builder().url(this@httpGetAsync).build())
```

Comment:
why did we change `suspendCall` to `call` invocation`?

Reply:
Changed back